lib/mbta.js

```diff
@@ -2,6 +2,66 @@
 
 // Code here.
 
+//Dev Notes:
+//Create a function that calculates the number of stops in between stations on the MBTA.
+//Write additional functions as needed.
+
+
+//Not gonna lie, I'm feeling pretty lost on this one.
+
+
+//First, we will define each of the lines on the MBTA as an object.
+let redLine = [
+  {station: "South Station"},
+  {station: "Park Street"},
+  {station: "Kendall"},
+  {station: "Central"},
+  {station: "Harvard"},
+  {station: "Porter"},
+  {station: "Davis"},
+  {station: "Alewife"},
+]
+
+let greenLine = [
+  {line: "Green", station: "Government Center"},
+  {line: "Green", station: "Park Street"},
+  {line: "Green", station: "Boylston"},
+  {line: "Green", station: "Arlington"},
+  {line: "Green", station: "Copley"},
+  {line: "Green", station: "Hynes"},
+  {line: "Green", station: "Kenmore"},
+]
+
+let orangeLine = [
+  {line: "Orange", station: "North Station"},
+  {line: "Orange", station: "Haymarket"},
+  {line: "Orange", station: "Park Street"},
+  {line: "Orange", station: "State"},
+  {line: "Orange", station: "Downtown Crossing"},
+  {line: "Orange", station: "Chinatown"},
+  {line: "Orange", station: "Back Bay"},
+  {line: "Orange", station: "Forest Hills"},
+]
+
+//We want the user to have the ability to input their start and end locations, so we'll create empty variables for them now.
```

Comment:
You don't actually need to initialize variables like this, since users will be creating variables when they pass arguments into a `stopsBetweenStations` function. For example, if you have a function like this: `const consoleLogString = function(randomString){console.log(randomString}` as you can see, you are able to access the argument `randomString` inside the function